src/features/utils/instrument-base.js

```diff
@@ -14,28 +14,32 @@ export class InstrumentBase extends FeatureBase {
     this.hasAggregator = false
     this.auto = auto
 
+    /** @type {Function} This should be set by any derived Instrument class if it has things to do when feature fails or is killed. */
+    this.abortHandler;
+
     if (auto) registerDrain(agentIdentifier, featureName)
   }
 
+  /** This is responsible for pulling in and executing the latter part of the feature--its aggregator. The first part--the instrumentation--should call this at the end of its setup. */
   importAggregator() {
-    try {
-      if (this.hasAggregator || !this.auto) return
-      this.hasAggregator = true
-      const lazyLoad = async () => {
-        try {
-          const { Aggregate } = await import(`../../features/${this.featureName}/aggregate`)
-          new Aggregate(this.agentIdentifier, this.aggregator)
-          this.resolve()
-        } catch (e) {
-          warn(`Failed to import aggregator class for ${this.featureName}`, e)
-        }
+    if (this.hasAggregator || !this.auto) return
+    this.hasAggregator = true
+    const waitForWindowLoad = async () => {
```

Comment:
This name change seems to indicate that this function will wait for the window to load.  This is a misnomer, and i think it made more sense the way it was before (lazyLoad some module when the function is called)